<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSVColumnConfigurationDialog</class>
 <widget class="QDialog" name="CSVColumnConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select the Role of the Column</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Column role</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="10" column="0">
       <widget class="QRadioButton" name="ignoreRB">
        <property name="text">
         <string>Ignore this column</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QRadioButton" name="startRB">
        <property name="text">
         <string>Annotation start position</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QCheckBox" name="startOffsetCheck">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Add offset</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="startOffsetValue">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="suffix">
         <string> bp</string>
        </property>
        <property name="minimum">
         <number>-1000000000</number>
        </property>
        <property name="maximum">
         <number>1000000000</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="endRB">
        <property name="text">
         <string>Annotation end position</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="endInclusiveCheck">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Inclusive</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="lengthRB">
        <property name="text">
         <string>Annotation length</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QRadioButton" name="complMarkRB">
        <property name="text">
         <string>Complement strand mark</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="complValueCheck">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Mark value</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="5">
       <widget class="Line" name="line">
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QRadioButton" name="nameRB">
        <property name="text">
         <string>Annotation name</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QRadioButton" name="qualifierRB">
        <property name="text">
         <string>Qualifier</string>
        </property>
       </widget>
      </item>
      <item row="8" column="3" colspan="2">
       <widget class="QLineEdit" name="qualifierNameEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="5">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QLineEdit" name="complValueEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>250</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="groupRB">
        <property name="text">
         <string>Annotation group</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>134</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>qualifierRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>qualifierNameEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>endRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>endInclusiveCheck</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startOffsetCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>startOffsetValue</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>complValueCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>complValueEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CSVColumnConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CSVColumnConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>212</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
